tests/src/Interop/IJW/CopyConstructorMarshaler/IjwCopyConstructorMarshaler.cpp

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma managed
+class A
+{
+    int copyCount;
+    int i;
+
+public:
+    A()
+    :copyCount(0)
+    {};
+
+    A(A& other)
```

Comment:
Can you also add a test for a copy constructor from a class that has a base class?